mods/ra/rules/infantry.yaml

```diff
@@ -706,7 +705,6 @@ Ant:
 	Mobile:
 		Speed: 99
 		TurnSpeed: 12
-		SharesCell: no
```

Comment:
Related to #15235, which is not fixed here.  This PR just removes the bogus definition.